### PR TITLE
0.6.0-alpha1 release

### PR DESCRIPTION
**Pull request by uint**
Need a new release of `voting-contract` to implement open text proposals in tgrade.

### CODE REVIEW

**Comment by uint** [comment]
The CI error is unrelated: #58

File: contracts/tg4-mixer/Cargo.toml [hunk 1]
```diff
@@ -46,8 +46,8 @@ cw-multi-test = { version = "0.11.0" }
 cosmwasm-schema = { version = "1.0.0-beta" }
 # version's workaround for issue with cyclic dependencies during cargo publish
 # https://github.com/rust-lang/cargo/issues/4242
-tg4-engagement = { path = "../tg4-engagement", version = ">= 0.5.3-2, < 1.0.0", features = ["library"] }
-tg4-stake = { path = "../tg4-stake", version = "0.5.4", features = ["library"] }
+tg4-engagement = { path = "../tg4-engagement", version = ">= 0.5.3-2, <= 0.6.0-alpha1", features = ["library"] }
```
Comment (maurolacy) [inline contracts/tg4-mixer/Cargo.toml:49]:
```suggestion
tg4-engagement = { path = "../tg4-engagement", version = ">= 0.5.5, <= 0.6.0-alpha1", features = ["library"] }
```
?

**Review by maurolacy — APPROVED** [review]
LGTM.

File: contracts/tg4-mixer/Cargo.toml (hunk 1 above)
Comment (uint) [reply contracts/tg4-mixer/Cargo.toml:49]: Sure.

**Comment by uint** [comment]
Rebased to include the CI fix from `main`.

**Pull request merged**